e2e/config/config.go

```diff
@@ -1,26 +1,73 @@
 package config
 
 import (
+	"crypto/ecdsa"
 	"errors"
 	"fmt"
 	"os"
 
 	"github.com/btcsuite/btcd/chaincfg"
-	"gopkg.in/yaml.v2"
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+	"gopkg.in/yaml.v3"
 )
 
+// DoubleQuotedString forces a string to be double quoted when marshaling to yaml.
+// This is required because of https://github.com/go-yaml/yaml/issues/847
+type DoubleQuotedString string
+
+func (s DoubleQuotedString) MarshalYAML() (interface{}, error) {
+	return yaml.Node{
+		Value: string(s),
+		Kind:  yaml.ScalarNode,
+		Style: yaml.DoubleQuotedStyle,
+	}, nil
+}
+
+func (s DoubleQuotedString) String() string {
+	return string(s)
+}
+
+func (s DoubleQuotedString) AsEVMAddress() (ethcommon.Address, error) {
+	if !ethcommon.IsHexAddress(string(s)) {
+		return ethcommon.Address{}, fmt.Errorf("invalid evm address: %s", string(s))
+	}
+	return ethcommon.HexToAddress(string(s)), nil
+}
+
 // Config contains the configuration for the e2e test
 type Config struct {
-	Accounts    Accounts  `yaml:"accounts"`
-	RPCs        RPCs      `yaml:"rpcs"`
-	Contracts   Contracts `yaml:"contracts"`
-	ZetaChainID string    `yaml:"zeta_chain_id"`
+	Accounts           Accounts           `yaml:"accounts"`
+	AdditionalAccounts AdditionalAccounts `yaml:"additional_accounts"`
+	RPCs               RPCs               `yaml:"rpcs"`
+	Contracts          Contracts          `yaml:"contracts"`
+	ZetaChainID        string             `yaml:"zeta_chain_id"`
+}
+
+// Account contains configuration for an account
+type Account struct {
+	RawBech32Address DoubleQuotedString `yaml:"bech32_address"`
+	RawEVMAddress    DoubleQuotedString `yaml:"evm_address"`
+	RawPrivateKey    DoubleQuotedString `yaml:"private_key"`
 }
 
-// Accounts contains the configuration for the accounts
+// Accounts are the required accounts to run any e2e test
 type Accounts struct {
-	EVMAddress string `yaml:"evm_address"`
-	EVMPrivKey string `yaml:"evm_priv_key"`
+	Deployer Account `yaml:"deployer"`
+}
```

Comment:
Maybe this should just be a top level value. Deployer isn't a great name anyway.

Maybe `DefaultAccount Account`

Reply:
Yes we should rename it